src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,35 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+
+        //Excavator excavator = new Excavator();
+        //Bulldozer bulldozer = new Bulldozer();
+        //Truck truck = new Truck();
+
+        Object[] object = new Object[3];
+
+        object[0] = new Excavator();
+        object[1] = new Bulldozer();
+        object[2] = new Truck();
+
+
+        for (Object objects : object ) {
```

Comment:
The loop variable should have a singular name to improve readability, such as 'machine' instead of 'objects'.